pkg/apis/openebs.io/v1alpha1/types.go

```diff
@@ -142,3 +150,77 @@ type Task struct {
 	// Kind is the kind corresponding to the task's actual content
 	Kind string `json:"kind"`
 }
+
+// +genclient
+// +genclient:noStatus
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient:nonNamespaced
+// +resource:path=cstorpool
+
+// CStorPool describes a cstor pool resource created as custom resource.
+type CStorPool struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata",omitempty`
```

Comment:
Double quotes should come at the end for whole json tag like `json:"metadata,omitempty"`

Reply:
will change